source/install.html.haml

```diff
@@ -51,7 +51,7 @@ title: Install Sass
         %span.windows-icon
         %span.linux-icon
       %li
-        = link_to "Scout", "http://mhs.github.io/scout-app/"
+        = link_to "Scout", "http://scout-app.io/"
```

Comment:
The app is officially called "Scout-App" now, not just "Scout".

**Source:** I'm creator of Scout-App 2! :D